app/utils/External.ts

```diff
@@ -20,40 +20,61 @@ Text form of class hierarchy diagram to be used at: https://yuml.me/diagram/nofu
 
 */
 
-function getContentType(type) {
+interface RequestOptions {
+    params?: Record<string, any>;
+    data?: any;
```

Comment:
How about using `body` as the name? `body` is a bit more specific than `data` IMO :slightly_smiling_face: 

```suggestion
    body?: any;
```

Reply:
I named it after the pre-existing parameter name, but you are right - updated